extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -46,6 +49,57 @@ export class ProjectsController {
 		this.buildHelper = new BuildHelper();
 	}
 
+	public get DeployInfo(): (dataworkspace.IDashboardColumnData | dataworkspace.IDashboardColumnDataGroup)[][] {
```

Comment:
it's a little confusing that there are classes named BuildInfo and DeployInfo and then there are these functions that return the build and deploy infos transformed into IDashboardColumnData. Either rename this to something more descriptive or move this logic to a function in BuildInfo and DeployInfo like getDashBoardColumnData()

Reply:
Updated the get methods to get dashboardDeployData() and get dashboardBuildData().